Source/PublicAPI/EThree+Authorization.swift

```diff
@@ -73,14 +75,24 @@ extension EThree {
         }
     }
 
+#if os(iOS)
+    /// Updates keychain entries with new options
+    ///
+    /// - Parameter set: private key will be accessed only using biometrics or passcode if true
+    /// - Throws: corresponding error
+    @objc public func setBiometricProtection(to set: Bool) throws {
```

Comment:
Bad argument naming

Reply:
What's the use case for this method?

Reply:
If user wants to stop using biometric protection for key, he needs some method to resave key with new options. Apps with biometric protection provide such functionality